torch/csrc/jit/autodiff.cpp

```diff
@@ -246,15 +257,15 @@ class GradientHelper {
       auto graph = node->owningGraph();
       auto backward_value = graph->insert(
           aten::native_batch_norm_backward,
-          {grads.at(0).value(),
-           inputs.at(0).value(),
-           inputs.at(1).value(),
-           inputs.at(3).value(),
-           inputs.at(4).value(),
-           outputs.at(1).value(),
-           outputs.at(2).value(),
-           inputs.at(5).value(),
-           inputs.at(7).value(),
+          {grad_values.at(0),
+           inputs.at(0),
```

Comment:
Btw is it a good idea to have some assertions/checks for these lengths before accessing them?

Reply:
Per offline conversation with @ailzhang: can this function be moved to symbolic script? is it even necessary anymore? 

It might be interesting to handle in a follow-up PR. This one is focused on removing SymbolicVariable.